### DISCUSSION TITLE
Incomplete WSL installation - need help

### DISCUSSION THREAD

**Discussion by dlin95123** [microsoft/WSL#11486] [Q&A]
I installed WSL2 on my new Win11 Pro laptop and installed Ubuntu distribution, using the wsl --install command in PowerShell. The Ubuntu Linux appears to work fine. But when I did wsl --update in PowerShell, it failed at 90% and showed an error:

Element not found

Then I installed Docker Desktop and the Docker Engine stopped at launch with the following error message:

![image](https://github.com/microsoft/WSL/assets/103467788/2dc8205e-8bbf-438f-89f9-bc74cbce5be3)

Please advise how I can go about to fix the problems.

**Comment by Yewzir** [comment]
@dlin95123 

Please reinstall Docker Desktop **4.28.0** for Windows, and try again.

**Reply by picarro-dlin** [reply]
Thanks. I re-installed Windows and other apps. It seems to be fine after that.

**Reply by Yewzir** [reply]
Great to hear. Your issue seemed to have affected a wide range of users, though.

**Reply by picarro-dlin** [reply]
I have 3 computers... only 1 had that problem....

**Reply by Yewzir** [reply]
But billions of people in the world.
